rr_viz/src/ross/ross_robotics.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
is there any particular reason for naming this file this way?

Reply:
Nope, I just picked a name for it. Its the main script that has all of our specific tabs. I've given it a better name